common/src/main/java/gov/cms/ab2d/common/dto/PropertiesDTO.java

```diff
@@ -0,0 +1,17 @@
+package gov.cms.ab2d.common.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+public class PropertiesDTO {
+
+    private Long id;
```

Comment:
Since DTO is what flows through the API to the end user, we don't need to expose internal primary key (`id`).